cmake/Modules/BuildSettings.cmake

```diff
@@ -18,7 +18,7 @@ set(CMAKE_CXX_COMPILE_FEATURES
 #
 set(CMAKE_INSTALL_MESSAGE LAZY)
 set(CMAKE_C_STANDARD 11 CACHE STRING "C language standard")
-set(CMAKE_CXX_STANDARD 11 CACHE STRING "CXX language standard")
+set(CMAKE_CXX_STANDARD 14 CACHE STRING "CXX language standard")
```

Comment:
Why is this change necessary?

Reply:
We were running into segfaults and you found that it was fixed by setting the standard to 14. This was what you told me:

"I was playing with your build, and I observed that both cxx and c gridrec work for Linux if the CXX_STANDARD is set correctly (it must be at least 14 to enable std::complex). The accelerated functions cause a segmentation fault."

Reply:
Hmmm... I don't trust past me. Let's recheck this.

Reply:
I think past you was right. When building on Windows, I run into errors when the CXX_STANDARD is 11 when building gridrec. Setting the standard to 14 fixes the issue.